### PR TITLE
AEA-1381: Map high repeat number from two possible locations.

### PR DESCRIPTION
**Pull request by mfjarvis**

### CODE REVIEW

File: coordinator/src/services/translation/response/cancellation/cancellation-medication-request.ts [hunk 1]
```diff
@@ -156,7 +155,7 @@ function getPrescriptionStatusInformation(code: string, display: string) {
     case "5000":
       return {
         prescriptionStatusCode: "R-5000",
-        prescriptionStatusDisplay: `Unable to process message.${extraInformation}`,
+        prescriptionStatusDisplay: display,
```
Comment (mfjarvis) [inline coordinator/src/services/translation/response/cancellation/cancellation-medication-request.ts:158]: Irrelevant to this PR but harmless - making our error handling consistent.

File: coordinator/tests/services/translation/request/prescription.spec.ts [hunk 2]
```diff
@@ -432,6 +434,91 @@ describe("createRepeatNumberForMedicationRequests", () => {
   })
 })
 
+describe("extractRepeatNumberHighValue", () => {
+  test("extracts from dispenseRequest", () => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```
Comment (dan-jones-iw) [inline coordinator/tests/services/translation/request/prescription.spec.ts:440]: maybe this is the best way to do these kinds of tests :joy: bit painful to look at tho. could we not do some message/resource cloning from somewhere?

File: coordinator/tests/services/translation/request/prescription.spec.ts [hunk 3]
```diff
@@ -432,6 +434,91 @@ describe("createRepeatNumberForMedicationRequests", () => {
...
+    const testMedicationRequest: fhir.MedicationRequest = {
+      extension: [{
+        url: "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-MedicationRepeatInformation",
+        extension: []
+      }],
+      dispenseRequest: {}
+    }
+    expect(() =>extractRepeatNumberHighValue(testMedicationRequest)).toThrow(InvalidValueError)
```
Comment (dan-jones-iw) [inline coordinator/tests/services/translation/request/prescription.spec.ts:481]: whitespace

**Review by dan-jones-iw — APPROVED** [review]

File: coordinator/tests/services/translation/request/prescription.spec.ts (hunk 2 above)
Comment (mfjarvis) [reply coordinator/tests/services/translation/request/prescription.spec.ts:440]: Wouldn't be so ugly if the linter allowed me to use `as` :wink:

Comment (mfjarvis) [reply coordinator/tests/services/translation/request/prescription.spec.ts]: Managed to appease the linter. See what you think now

File: coordinator/tests/services/translation/request/prescription.spec.ts (hunk 3 above)
Comment (mfjarvis) [reply coordinator/tests/services/translation/request/prescription.spec.ts]: How embarrassing. Sorted

File: coordinator/tests/services/translation/request/prescription.spec.ts (hunk 2 above)
Comment (dan-jones-iw) [reply coordinator/tests/services/translation/request/prescription.spec.ts]: much better :+1:

**Review by dan-jones-iw — APPROVED** [review]

**Pull request merged**